src/main/java/ee/tuleva/onboarding/mandate/application/ApplicationService.java

```diff
@@ -104,6 +105,18 @@ public boolean hasPendingWithdrawals(Person person) {
         .collect(toList());
   }
 
+  private FundDto getTargetFund(MandateFundsTransferExchangeDTO exchangeDTO, String language) {
+    String targetFundIsin = exchangeDTO.getTargetFundIsin();
+    if (targetFundIsin == null) return null;
```

Comment:
Why return null? This might propagate into the system and cause a `NullpointerException` from somewhere else.

Reply:
Is is null when there's a `targetPik` instead?

Reply:
Would be great to have some validation for this somewhere - i.e. in the `TransferApplicationDetails.Exchange` constructor. So it would quickly throw an exception if something's wrong. We're effectively doing an [XOR](https://www.maximintegrated.com/content/dam/images/glossary/xor-gate-symbol.jpg) on `targetFund` and `targetPik`. *One and only one* should be filled. Not both. Not neither.